pkg/config/template.go

```diff
@@ -1321,6 +1321,7 @@ const templateStringCrioRuntimeRuntimesRuntimeHandler = `# The "crio.runtime.run
 {{ $.Comment }}runtime_path = "{{ $runtime_handler.RuntimePath }}"
 {{ $.Comment }}runtime_type = "{{ $runtime_handler.RuntimeType }}"
 {{ $.Comment }}runtime_root = "{{ $runtime_handler.RuntimeRoot }}"
+{{ $.Comment }}use_default_runtime = {{ $runtime_handler.UseDefaultRuntime }}
```

Comment:
Can we add a section above to explain the feature, like which values are inherited from the default runtime?

Reply:
Done.